frontend/src/routes/trade/AddEvidence.svelte

```diff
@@ -0,0 +1,127 @@
+                Which articles helped you decide on your position?
+            </header>
+            <div
+                class="block"
+                style="height: 50vh; overflow-x: hidden; overflow-y: auto;">
+                <div class="tile is-ancestor px-2">
+                    {#await newsRequest}
+                        <div
+                            style="width: 100%; height: 5vh; display: flex; justify-content: center; align-items: center;">
+                            <p>Loading...</p>
+                        </div>
+                    {:then responses}
+                        <div class="tile is-parent is-vertical">
+                            {#each responses as response}
+                                <NewsTile
+                                    {toggleArticle}
+                                    data={response}
+                                    selected={articleEvidence.includes(
+                                        response.article_id,
+                                    )} />
+                            {/each}
+                        </div>
+                    {:catch error}
+                        <div
+                            style="width: 100%; height: 5vh; display: flex; justify-content: center; align-items: center;">
+                            <p>{error.message}</p>
+                        </div>
+                    {/await}
+                </div>
+            </div>
+        </div>
+        <div class="column is-half px-5">
+            <header class="title is-4">
+                Are there any relevant financial statistics?
+            </header>
+
+            <div class="columns mb-5">
+                <div class="column">
+                    <label class="checkbox">
+                        <input type="checkbox" />
+                        Remember me
+                    </label>
+                </div>
+                <div class="column">
+                    <label class="checkbox">
+                        <input type="checkbox" />
+                        Remember me
+                    </label>
+                </div>
+                <div class="column">
+                    <label class="checkbox">
+                        <input type="checkbox" />
+                        Remember me
+                    </label>
+                </div>
+                <div class="column">
+                    <label class="checkbox">
+                        <input type="checkbox" />
+                        Remember me
+                    </label>
+                </div>
+            </div>
+            <br />
+            <br />
+            <div class="block">
+                <header class="title is-4">
+                    Why does this evidence justify your position?
+                </header>
+                <textarea
+                    class="textarea"
+                    style="height: 24vh;"
+                    value={textEvidence}
```

Comment:
Maybe some `placeholder="Share your insights..."` to make it clearer at first glance what this block is intended for?

Reply:
Good idea :)